### ISSUE TITLE
MA0005 shouldn't be reported

### ISSUE THREAD

**Issue by grosch-intl** [meziantou/Meziantou.Analyzer#1530]
### Version of the Meziantou.Analyzer NuGet package

3.0.258

### Rule Identifier

MA005

### Target Framework

net10

### C# Language version

13

### Description

The rule shows up on the below code saying to use Array.Empty<T> instead.

### Reproduction Steps

Minimal code:

```c#
public static TheoryData<string> Groups => [ "foo", "bar" ];
```

**Comment by meziantou** [comment]
Roslyn changed the behavior of collection expressions in version 5.6. Will make a fix for it.
